terraform/gcp/modules/iam/main.tf

```diff
@@ -0,0 +1,36 @@
+# Create Google Service Account
+resource "google_service_account" "service_account" {
+  account_id   = format("%s-sa", var.name)
+  display_name = "GKE Security Service Account"
+  project      = var.project_id
+}
+
+resource "google_service_account_key" "service_account_key" {
```

Comment:
We're not going to use service accounts, only workload identity, so please remove